draft-documents/ipld_did_documents.md

```diff
@@ -135,7 +135,6 @@ This smart contract has the added benefit of listening to updates to the IPLD di
 
 Example 1 shows the DID document before publication, and before a cryptographic signature has been added. Note that the `id` and `signature` fields are omitted as it is not associated yet with the DID method specific identifier.  Unlike other DID methods, with the DID document being represented as IPLD, we can directly link the `@context` which is also represented on IPLD as a resolvable CID cryptographic link.  Noteably absent is the `previous` field, as this is the genesis of the chain of objects that subsequent updates will reference (see below). This entire DID document when added to IPFS as IPLD has a CID of `zdpuAqiExr6k4AbWF6BuGkgUbVMZ7jbJyNvRz9z9yyRBxosPi` and will be used as the `previous` field in the subsequent updated DID document (below).  
```

Comment:
The last sentence of the paragraph on line 136 is still a little confusing; from the phrasing, I expect to see that as `previous` in Example 2. However, your subsequent text clarifies how it fits together. I suggest a slight rewording of the last sentence (see bolded area)

This entire DID document when added to IPFS as IPLD has a CID of `zdpuAqiExr6k4AbWF6BuGkgUbVMZ7jbJyNvRz9z9yyRBxosPi` and will be used as the `previous` field in the subsequent updated DID document **`zdpuAvqt1YTeAdcyyncBDdJLVgKyZNbHm17rBEJNqFzVbg24B` (discussed below).**